tasks/extensions/dev_headers.yml

```diff
@@ -10,14 +10,56 @@
   notify:
     - restart postgresql
 
+# This likley only applies to version 11+ but currently can not build for anything below 12
+# Once that is restored, I will verify and add an addtional filter base on postgresql version
+- name: PostgreSQL | Extensions | Install Additonal Repositories | CentOS
+  yum:
+    name: "{{ item }}"
+    state: present
+  with_items:
+    - centos-release-scl
+    - epel-release
+  when:
+    - ansible_distribution == "CentOS"
+
+- name: PostgreSQL | Extensions | Install Additonal Repositories | RockyLinux
+  block:
+
+  - name: PostgreSQL | Extensions | Install EPEL | RockyLinux
+    yum:
+      name: "{{ item }}"
+      state: present
+    with_items:
+      - epel-release
+
+  - name: PostgreSQL | Extensions | Check for enabled repos | RockyLinux
+    command: yum repolist
+    changed_when: false
+    register: _yum_repolist
+
+  - name: PostgreSQL | Extensions | Enable powertools | RockyLinux 8
+    command: yum -y config-manager --set-enabled powertools
+    when:
+       - "'powertools' not in _yum_repolist.stdout"
+       - ansible_distribution_major_version == "8"
+
+  - name: PostgreSQL | Extensions | Enable crb | RockyLinux 9
+    command: yum -y config-manager --set-enabled crb
+    when:
+       - "'crb' not in _yum_repolist.stdout"
+       - ansible_distribution_major_version == "9"
+
+  when:
+    - ansible_distribution == "Rocky"
+
 - name: PostgreSQL | Extensions | Make sure the development headers are installed | RedHat
-  yum: 
+  yum:
```

Comment:
Can we just use “package” ansible module instead of “yum” and “dnf”?

Reply:
yes!  I improved this and retested